packages/payment-processor/src/contracts/Erc20Contract.ts

```diff
@@ -1,31 +1,22 @@
-import { Contract, Signer } from 'ethers';
-import { Provider } from 'ethers/providers';
-import { BigNumber, BigNumberish, Interface } from 'ethers/utils';
-import { ITypedFunctionDescription } from './TypedFunctionDescription';
+import { Contract, Signer, providers } from 'ethers';
```

Comment:
Important. It became very hard with v5 to strongly type the contracts, so I simply removed the strong typing. :(

Reply:
ERC20 are so well known. I feel comfortable with that.